lib/option.rb

```diff
@@ -138,6 +138,13 @@ def display_name
 
   POSTGRES_VERSION_OPTIONS = ["17", "16"].freeze
 
+  PostgresHaOption = Data.define(:name, :standby_count, :description)
+  POSTGRES_HA_OPTIONS = [
```

Comment:
Since we use it with `.find {}` all the time, why don't we keep it as a `name => PostgresHaOption` hash?

Reply:
Pretty much every place you are calling `find { it.name == name }` on an array constant, you should probably be using a hash instead. If you really need a separate array constant: `ARRAY_CONSTANT = HASH_CONSTANT.keys.freeze`